wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -1073,4 +1083,24 @@ private void updateSyncPaneVisibility(int id, boolean visible) {
         finish();
         return Unit.INSTANCE;
     };
+
+    @Override
+    public void onLockScreenDeactivated() {
+        if (config.getShowNotificationsExplainer()) {
+            explainPushNotifications();
+        }
```

Comment:
If it's an update, push explainer dialog shows up when the user unlocks the device.